experiment_manager.py

```diff
@@ -334,6 +345,9 @@ def one_dimensional_job(self, config: Config):
 
         job = config.backend.value.run(circs)
         self._async_results.append(job)
+        print('1D job sent in:')
+        toc()
+        print('')
```

Comment:
I am confused.

Reply:
this is what it does:
![image](https://user-images.githubusercontent.com/67901011/183855388-8300c856-b30e-476c-93a5-3ff9c45a535b.png)